wgsl/index.bs

```diff
@@ -7301,8 +7301,18 @@ format to channel format.
 
 **Returns:**
 
-If all the parameters are within bounds, the unfiltered texel data.<br>
-If any of the parameters are out of bounds, then zero in all components.
+The unfiltered texel data.
+
+An out-of-bounds access occurs if:
+* any element of `coords` is outside the range `[0, textureDimensions(t))`
```

Comment:
I think this should take the level into account as well.

Reply:
that's the third bullet?